launch.ks

```diff
@@ -3,7 +3,7 @@
 // Launch.ks just try to decide if is better to use
 // launch_asc.ks or launch_ssto.ks
 /////////////////////////////////////////////////////
-parameter Apo is 200000.
+parameter Apo is 200. //km
```

Comment:
I'm :-1: on this change.  Please keep it at meters, not kilometers.  All the documentation and example scripts which reference `launch_asc()` use meters.  Imagine what happens when someone tries this:  `run launch_asc(body:atm:height + (body:radius / 10)).`

Reply:
I again disagree, but will again follow established practices and revert that.